docs/REST API/rest-api.md

```diff
@@ -8926,13 +8939,26 @@ Returns information about the specified software. By default, `versions` are sor
       "installer_id": 23,
       "team_id": 3,
       "uploaded_at": "2024-04-01T14:22:58Z",
+      "install": "automatic"
       "install_script": "sudo installer -pkg /temp/FalconSensor-6.44.pkg -target /",
       "pre_install_query": "SELECT 1 FROM macos_profiles WHERE uuid='c9f4f0d5-8426-4eb8-b61b-27c543c9d3db';",
       "post_install_script": "sudo /Applications/Falcon.app/Contents/Resources/falconctl license 0123456789ABCDEFGHIJKLMNOPQRSTUV-WX",
       "self_service": true,
+      "labels_exclude_any": [
+        {
```

Comment:
FYI @fleetdm/frontend, I'll add the following interface for this object, which the other Label interfaces will extend:
```
export interface ILabelIdentifier {
  id: number;
  name: string;
}
```